azurerm/provider.go

```diff
@@ -206,6 +206,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_lb_probe":                                                               resourceArmLoadBalancerProbe(),
 			"azurerm_lb_rule":                                                                resourceArmLoadBalancerRule(),
 			"azurerm_local_network_gateway":                                                  resourceArmLocalNetworkGateway(),
+			"azurerm_log_analytics_linked_service":                                           resourceArmLogAnalyticsLinkedService(),
```

Comment:
I agree the name should probably be `azurerm_log_analytics_workspace_linked_service`, thoughts @tombuildsstuff?

Reply:
I've renamed the resource. Will also change the description up above.